skatepark/sass/templates/_header.scss

```diff
@@ -0,0 +1,33 @@
+header {
+	justify-content: space-between; // Apply a cluster (flex?) layout
+	flex-wrap: wrap-reverse;
+	> * { 
+		flex-grow: 1;
+		> * { // Apply a stack layout (page 69 of the every-layout.dev PDF) 
+			margin-top: 20px;
+			margin-bottom: 20px;
+		}
+	}
+}
+
+@include break-small-only(){
+	.nav-links {
+		display: flex;
+		justify-content: space-between;
+		width: 100%;
+		border-bottom: 2px solid var(--wp--custom--color--primary);
+		.wp-block-navigation__responsive-container-open { 
+			display: flex;
+			align-items: center;
+
+			&::after {
+				content: "Menu";
```

Comment:
Could we use the lang attribute on the `html` tag to translate this? Something like:

```
html[lang="en-US"] {
	.nav-links {
		.wp-block-navigation__responsive-container-open {
			&::after {
				content: "Menu";
			}
		}
	}
}
```

I guess this would need to be added via a PHP template in an inline style.. Or is this horrible and hacky?

Reply:
we could add this line of CSS as inline inside functions.php and wrap the string around the PHP translation function directly, no need to target the lang attribute. It's not much better, but it's another option

Reply:
Ah nice ok! Well, we have options 😄

Reply:
Ideally there is an option in the nav block itself to provide this label next to the responsive hamburger. I wonder if it's planned